vsphere_plugin_common/__init__.py

```diff
@@ -2706,11 +2714,11 @@ def create_storage(self, vm_id, storage_size, parent_key, mode,
             if isinstance(vm_device, vim.vm.device.VirtualDisk):
                 # Generate filename (add increment to VMDK base name)
                 vm_disk_filename_cur = vm_device.backing.fileName
-                p = re.compile('^(\[.*\]\s+.*\/.*)\.vmdk$')
+                p = re.compile('^(\\[.*\\]\\s+.*\\/.*)\\.vmdk$')
```

Comment:
What is this change for? Can you add a comment to the code showing an expected filename example?